waku/v2/waku_rln_relay/rln/wrappers.nim

```diff
@@ -51,30 +53,69 @@ proc membershipKeyGen*(ctxPtr: ptr RLN): RlnRelayResult[IdentityCredential] =
 
   return ok(identityCredential)
 
-proc createRLNInstanceLocal*(d: int = MerkleTreeDepth): RLNResult =
+type RlnTreeConfig* = ref object of RootObj
+  cache_capacity*: int
+  mode*: string
+  compression*: bool
+  flush_interval*: int
+  path*: string
+
+type RlnConfig* = ref object of RootObj
+  resources_folder*: string
+  tree_config*: RlnTreeConfig
+
+proc `%`(c: RlnConfig): JsonNode =
+  ## wrapper around the generic JObject constructor.
+  ## We don't need to have a separate proc for the tree_config field
+  let tree_config = %{ "cache_capacity": %c.tree_config.cache_capacity,
+                       "mode": %c.tree_config.mode,
+                       "compression": %c.tree_config.compression,
+                       "flush_interval": %c.tree_config.flush_interval,
+                       "path": %c.tree_config.path }
+  return %[("resources_folder", %c.resources_folder),
+           ("tree_config", %tree_config)]
+
+proc createRLNInstanceLocal*(d = MerkleTreeDepth, 
```

Comment:
This can be "private" I guess :)
```suggestion
proc createRLNInstanceLocal(d = MerkleTreeDepth, 
```

Reply:
Thanks, addressed in [f8774ee](https://github.com/waku-org/nwaku/pull/1782/commits/f8774ee0ec9a7d62bcef103c5b86d35b507cdb7d)